packages/office-addin-debugging/src/start.ts

```diff
@@ -173,41 +173,123 @@ export async function runPackager(commandLine: string, host: string = "localhost
 }
 
 /**
- * Start debugging
- * @param manifestPath The path to the manifest file.
- * @param appType The type of application to debug.
- * @param app The Office application to debug.
- * @param debuggingMethod The method to use when debugging.
- * @param sourceBundleUrlComponents Specify components of the source bundle url.
- * @param devServerCommandLine If provided, starts the dev server.
- * @param devServerPort If provided, port to verify that the dev server is running.
- * @param documentUrl If provided, the document to open for sideloading to web.
- * @param packagerCommandLine If provided, starts the packager.
- * @param packagerHost Specifies the host name of the packager.
- * @param packagerPort Specifies the port of the packager.
- * @param enableDebugging If false, start without debugging.
+ * `startDebugging` options.
  */
-export async function startDebugging(manifestPath: string, appType: AppType, app: OfficeApp | undefined,
-    debuggingMethod: DebuggingMethod = defaultDebuggingMethod(),
+export interface StartDebuggingOptions {
+
+    /**
+     * The path to the manifest file.
+     */
+    manifestPath: string,
```

Comment:
Joe, based on our discussion earlier, should some of these be required parameters for the startDebugging() function instead of in the options object?

Reply:
In this case, since there are so many options, I think it make sense to keep it as is.